src/dumpulator/dumpulator.py

```diff
@@ -1075,7 +1079,7 @@ def map_module(self, file_data: bytes, file_path: str = "", requested_base: int
             if write:
                 protect = MemoryProtect.PAGE_READWRITE
             if execute:
-                protect = MemoryProtect(protect.value << 4)
+                protect = MemoryProtect.PAGE_EXECUTE_READ

```

Comment:
There is no `elif` here, the `<< 4` is converting from PAGE_XX to PAGE_EXECUTE_XX (which I agree warrants a comment)

![image](https://user-images.githubusercontent.com/2458265/221174263-691dd7e4-5f0c-4fc1-8ea7-88f4e42bf5e3.png)

Reply:
Mistake on my end.

Reply:
Fixed by #[6c18ee3](https://github.com/mrexodia/dumpulator/pull/47/commits/6c18ee378cc70ffad68b185cf6e50f0ea9682849)